core/src/test/java/io/confluent/rest/JsonParseExceptionMapperTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.rest;
+
+import com.fasterxml.jackson.core.JsonLocation;
+import com.fasterxml.jackson.core.JsonParseException;
+import io.confluent.rest.entities.ErrorMessage;
+import io.confluent.rest.exceptions.JsonParseExceptionMapper;
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
These are junit 5, and I don't think 5.4.x is set up for junit 5.  Change to the junit 4 equivalent.